.travis.yml

```diff
@@ -3,29 +3,22 @@ sudo: false
 cache:
   pip: true
 python:
-- '3.5'
-deploy:
-  provider: pypi
-  user:
-    secure: Jpkqo70FGa0K1+VGPIvgAisNa1FB/Hd4rcwgxJtYUQc3wby+nyxuIrzsu7NOsYkDJ8vxaE/FbKND1ZFW18vbD9kY6S3/oMSCE5f4fAEhpPEMlOGW5jLI00427PdhbYxk1X2NkhMpU5mEYFFoEf4+GsfoO6zKhhH/xVHjirrcKSJoThQpbYymvJ6uQAuNddSSxqDIcHJGDEfFPUqydOlqgi4oizCagl8N+gIJqpDQ6VQopN8tdM/4YavMaOF7yeU/XEYBQLQgldcjwIeP/X0hy3bLpphgI2MoZ2Y8mJjWF3sC6stKhIjXrUdcjodpzPTJSVuaio1FajTn0h6E1COa+lmckWtzdkGIgSmA6gwKh44nFQcDSV/8fD/or4xjc210V1tG12+MugOuI5q4NFmjbjNEsKw7KIoDqPGRdfE/HZOr30oLWTgaEqdTtG0SzdFfQPq84P80UGlcv68CgnrXtQGwXR08/H4NsuDhnjBKFXZq1afC7Tqw7JtlJyXQmJnmKElrXNaugJe4ru4RY+FlGCt44QZe/sozzFutzqWUfNUrDP2AP+saF7CiH6kD+tcPDioD18dJx0Y0E726Hu7O4wGZk6gl1AYV0LluXulDBO1x8/5vJtzolvC4AsXYWAhk+uUP4j1pO0Pmi+ahuEyyYdUQScMr+gNe0aa/JpzV2L0=
-  password:
-    secure: dP62awneMbu7jlpFmkQpwGqDxmqlOJorbWjh51BLOuPDc/LHOlI50vdLCwNLwfQFedKoP5LZIfWYBoy8wbXLhXpqZkgwwPvBesaUPuRONdxLqXCqM21d/1nmgIEIdwgFkcIKTRwTPK5LKygXg5iAstkrq+kcdv0Hwake4QE7ZtJEo2kTKPF8jkzHteeBfSB1n6GrQ6iQnkzMg6RiqjB3t0sdNRX/sKZqM5CdYzA523tDjrPTGTWyhVS++/TeNW5PZajZyVr2HKH5+OYp5Ik/8w0Q6EXM51ZuFrqTnv67kTCSIfXai6tpCauI3x3EiUVhkyznIXz2uC5i518GWQWwbuf+k4IKmhXtUe6uA+rVaCkdBOjBN6CyLGJwKaefCJw/RMgWKnqqrHr+6hfwKJctQVxDQcXpvP3+O+LkqBY3D1CYtSjAiini3ooKa2UQGGBhCe8HcPXsMwzPzefdv36RdGASg4X27qOTCKm4F+Jzo9LWF5K3AmYDyGUOXlpfSK1gug6DBrp3VsE3rNX3z8PRj5+LZhDXKg/K99FzM3lsoYypP/V5btoZ3rbUvklRueotSjIO2mohzX4aJ7cZt8hMuoABaXdyIEggWccPYCkOnBnJ0yiXp/PpiRdXeVF2G7eo5fRM/TWPAkvNUJpFvQBRCtvW3xmirY+7Rf79S8FntHM=
-  on:
-    tags: true
-  distributions: sdist bdist_wheel
-install:
-- pip install -U -r requirements-dev.txt
-- pip install -U -r requirements.txt
+- '3.6'
```

Comment:
`pyproject.toml` does not rule out 3.7, so you should probably have a 3.7 build as well (if you think it is necessary). If this is a mistake disallow 3.7 in `pyproject.toml`.